src/Documentation/resources/ideas_for_contributions.md

```diff
@@ -5,16 +5,18 @@ title: Ideas for Contributions
 
 # Ideas for Contributions
```

Comment:
I think we should delete this page altogether. Some of the original ideas have already been implemented, some are outdated. I'm afraid this list is rather misleading now.